client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -167,40 +292,19 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 											) }
 										</strong>
 									</p>
-									<Flex justify="start">
-										<FlexItem className="transaction-details-dispute-accept-modal__icon">
-											<Icon icon={ backup } size={ 24 } />
-										</FlexItem>
-										<FlexItem>
-											{ createInterpolateElement(
-												sprintf(
-													/* translators: %s: dispute fee, <em>: emphasis HTML element. */
-													__(
-														'Accepting the dispute marks it as <em>Lost</em>. The disputed amount will be returned to the cardholder, with a %s dispute fee deducted from your account.',
-														'woocommerce-payments'
-													),
-													getDisputeFeeFormatted(
-														dispute,
-														true
-													) ?? '-'
-												),
-												{
-													em: <em />,
-												}
-											) }
-										</FlexItem>
-									</Flex>
-									<Flex justify="start">
-										<FlexItem className="transaction-details-dispute-accept-modal__icon">
-											<Icon icon={ lock } size={ 24 } />
-										</FlexItem>
-										<FlexItem>
-											{ __(
-												'Accepting the dispute is final and cannot be undone.',
-												'woocommerce-payments'
-											) }
-										</FlexItem>
-									</Flex>
+
+									{ disputeAcceptAction.modalLines.map(
```

Comment:
This is really generic and tightly coupled to `disputeAcceptAction` – to understand this code, requires reading whole file. How can we reduce that load?

Reply:
It's like an interface for generically rendering a modal. I don't think we need that capability, we can hard-code each modal we want as a self-contained component.

Reply:
A side effect of this is that the tracks events both fire the same props, even though they are not relevant for both events. The tracks event name and the props should be nearby in the code (i.e. inline in the `recordEvent()` call) IMO.